wordpress-mobile/WordPress-Android
# Switch to upload key for release bundle signing

This PR introduces signing with upload key. The upload key is used to sign the release bundle that is uploaded to the Play Store. The local release universal apk is still signed with the master key in order to test with the production configuration.

##To Test:
1. Checkout a new branch from this one (name should be `release/<something>`).
2. Bump the `versionCode` in `build.gradle` and commit.
3. Pull the latest `mobile-secrets` repository.
4. Run `./gradlew prodDeps` to update the local configuration.
5. Run `bundle exec fastlane build_beta(upload_to_play_store: true)`.
6. Verify that the new binary is uploaded to the Play Store Artifact library.
7. Verify that the universal apk in your local artifacts folder can be installed on a device on top of a previous version and that Google Login works.

**Clean up**:
8. Delete the APK from the Play Store Artifact library. 

PR submission checklist:
